### ISSUE TITLE
Multiple Modules Not Nesting In Sidebar

### ISSUE THREAD

**Issue by infinisuite** [drake7707/Typescript-Editor#1]
![2015-10-08_15-48-34](https://cloud.githubusercontent.com/assets/7989672/10377988/5d3debf0-6dd4-11e5-929c-b5933ecc75de.png)

We've been testing Typescript-Editor for use in our project and have an issue with multiple modules. If you have multiple sequential module statements then the sidebar will not be updated properly. The following will not nest properly in the sidebar:

``` javascript
module Iscript {    
    export interface eventInfo {};    
}
module contactlist_user_applet {
...
}
```

However, a nested module will show in the sidebar properly:

``` javascript
module contactlist_user_applet {

      module Iscript {
                export interface eventInfo {};    
      }     
     ...
}

We can nest modules as a work-around, but it would be nice to have sequential modules work as well.

Also, nice implementation!


```

**Comment by drake7707** [comment]
This is probably related to https://github.com/Microsoft/TypeScript/issues/4289, I had to circumvent the normal behaviour in order to get it working. I'll take a look.

**Comment by drake7707** [comment]
Should be fixed now, you might want to CTRL+F5 to refresh the browser cache.

**Comment by infinisuite** [comment]
Works great now! Thanks!

On Fri, Oct 9, 2015 at 12:48 PM, drake7707 notifications@github.com wrote:

> Closed #1 https://github.com/drake7707/Typescript-Editor/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/drake7707/Typescript-Editor/issues/1#event-431614826.

## 

_Lee Mellinger_
President & CEO
AccuSuite, LLC
www.accusuite.com
Mobile: 1.734.531.8702
Skype: leemellinger
LinkedIn: linkedin.com/in/leemellinger
https://www.linkedin.com/in/leemellinger

_Business Solutions for Cloud 2.0_

NOTICE: The information contained in this e-mail, and in any accompanying
documents, may constitute confidential and/or legally privileged
information. The information is intended only for use by the designated
recipient. If you are not the intended recipient (or responsible for the
delivery of the message to the intended recipient), you are hereby notified
that any dissemination, distribution, copying, or other use of, or taking
of any action in reliance on this e-mail is strictly prohibited. If you
have received this e-mail communication in error, please notify the sender
immediately and delete the message from your system.
